src/mrack/outputs/pytest_mh.py

```diff
@@ -61,21 +63,19 @@ def create_mh_config(self):
                     logger.error(f"Host {host['name']} not found in the database.")
                     continue
 
-                cfgdom["hosts"].append(
-                    {
-                        "hostname": get_fqdn(host["name"], domain.get("name", "")),
-                        "os": {
-                            "family": get_os_type(host),
-                        },
-                        "role": host["role"],
-                        "ssh": {
-                            "host": get_external_id(
-                                provisioned_host, host, self._config
-                            ),
-                        },
-                        **host.get("pytest_mh", {}),
-                    }
-                )
+                hostcfg = {
+                    "hostname": get_fqdn(host["name"], domain.get("name", "")),
+                    "os": {
+                        "family": get_os_type(host),
+                    },
+                    "role": host["role"],
+                    "ssh": {
```

Comment:
Hi there should be used mrack function get_username(if the host is a windows machine.) and get_password.  see https://github.com/neoave/mrack/blob/main/src/mrack/outputs/pytest_multihost.py#L103

Reply:
Done. I'm not sure how to write unit test for it. @pvoborni is there a way to mock windows machine that would hit this code path?

Reply:
Set `os` to  startwith `win` or set `os_type` to `windows` for some of the hosts in mocked job metadata in the test.

Reply:
This can be added by provisioning provider, e.g. if the cloud environment generates the password. But I don't understand why you need it.  You can also set the password on job metadata level for the host.

Reply:
That said. The first 'host' is the DB host. So if the test is mocking the DB then it is the thing to set.